sdk/openai/openai-assistants/test/public/utils/recordedClient.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AssistantsClient, OpenAIKeyCredential } from "../../../src/index.js";
+import {
+  Recorder,
+  RecorderStartOptions,
+  assertEnvironmentVariable,
+} from "@azure-tools/test-recorder";
+import { Test } from "mocha";
+import { createTestCredential } from "@azure-tools/test-credential";
+import { ClientOptions } from "@azure-rest/core-client";
+import { AzureKeyCredential } from "@azure/core-auth";
+
+const envSetupForPlayback: Record<string, string> = {
+  ENDPOINT: "https://endpoint",
+  AZURE_CLIENT_ID: "azure_client_id",
+  AZURE_CLIENT_SECRET: "azure_client_secret",
+  AZURE_TENANT_ID: "88888888-8888-8888-8888-888888888888",
+  SUBSCRIPTION_ID: "azure_subscription_id",
```

Comment:
We need to add all environment variables including OPENAI_API_KEY here, otherwise, our recordings will have leaked secrets in them.